### DISCUSSION TITLE
How to create a DB relationships?

### DISCUSSION THREAD

**Discussion by pirum1ch** [drawdb-io/drawdb#322] [Q&A] [answered]
I didn't get, how to create a DB relationship? 
left panel has text only: "drag for create...", but what and where?

**Comment by 1ilit** [comment, marked as answer]
like this: https://imgur.com/a/tBPx1s8

**Reply by pirum1ch** [reply]
Thnks a lot! 
Really it wasn't clear, puzzled my brain while solving!
